x-pack/plugins/reporting/server/lib/create_worker.ts

```diff
@@ -23,18 +25,20 @@ export function createWorkerFactory<JobParamsType>(reporting: ReportingCore, log
   // Once more document types are added, this will need to be passed in
   return async function createWorker(queue: ESQueueInstance) {
     // export type / execute job map
-    const jobExecutors: Map<string, RunTaskFn<unknown>> = new Map();
+    const jobExecutors: Map<string, RunTaskFn> = new Map();
 
-    for (const exportType of reporting.getExportTypesRegistry().getAll() as Array<
-      ExportTypeDefinition<JobParamsType, unknown, unknown, RunTaskFn<unknown>>
-    >) {
+    for (const exportType of reporting
+      .getExportTypesRegistry()
+      .getAll() as ExportTypeDefinition[]) {
```

Comment:
Odd that it doesn't unwrap this type for us...

Reply:
7cf43d817a6